service/experiment-config.yaml

```diff
@@ -2,20 +2,21 @@
 # Unless you are a fuzzbench maintainer running this service, this
 # will not work with your setup.
 
-trials: 20
+trials: 10
 max_total_time: 82800  # 23 hours, the default time for preemptible experiments.
-cloud_project: fuzzbench
+# cloud_project: fuzzbench
 docker_registry: gcr.io/fuzzbench
-cloud_compute_zone: us-central1-c
-experiment_filestore: gs://fuzzbench-data
-report_filestore: gs://www.fuzzbench.com/reports
-cloud_sql_instance_connection_name: "fuzzbench:us-central1:postgres-experiment-db=tcp:5432"
-worker_pool_name: "projects/fuzzbench/locations/us-central1/workerPools/buildpool-e2-std-32"  # Mem 128 GB
-preemptible_runners: true
+#cloud_compute_zone: us-central1-c
+experiment_filestore: /tmp/experiment-data
+report_filestore: /tmp/report-data
```

Comment:
Did you forget to undo these changes?

Reply:
yes, sorry. i was doing local exp. but i forgot to revert it.
now i reverted it to origin/main